### ISSUE TITLE
Python 3.5 support

### ISSUE THREAD

**Issue by TinoDidriksen** [apertium/apertium-recursive#26]
https://docs.python.org/3/library/subprocess.html#subprocess.check_output "Changed in version 3.6: encoding and errors were added."

Supported distros Debian Stretch and Ubuntu Xenial are both on Python 3.5, so all Python code must be compatible with 3.5. There may be other issues beyond `subprocess.check_output(encoding=...)`.

**Comment by TinoDidriksen** [comment]
The tests now run, but these 2 fail on Ubuntu Xenial and Debian Stretch:
````
test_output (__main__.GlobalVars) ... ERROR
test_output (__main__.GlobalVarsClipping) ... ERROR

======================================================================
ERROR: test_output (__main__.GlobalVars)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./run_tests.py", line 26, in test_output
    actual = subprocess.check_output(args, input=self.input, universal_newlines=True)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['../src/rtx-proc', '-a', 'GlobalVars.bin']' returned non-zero exit status -11

======================================================================
ERROR: test_output (__main__.GlobalVarsClipping)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./run_tests.py", line 26, in test_output
    actual = subprocess.check_output(args, input=self.input, universal_newlines=True)
  File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 708, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['../src/rtx-proc', '-a', 'GlobalVarsClipping.bin']' returned non-zero exit status -11
````

**Comment by TinoDidriksen** [comment]
Yup, fixed in 39ed56d78cebebb6dd7fbb9d4ea76653925e9dfc - and packaged in Nightly for all platforms except Windows.
